### DISCUSSION TITLE
ElasticSearch Mappings Missing

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8425] [Unsupported Versions]
Finding the new SOC thing to be _far_ less useful for domain-specific hunting activities than the ye olde Kibana dashboards, but looks like the kibana dashboards aren't matching the new ES schema:
![sip](https://user-images.githubusercontent.com/1331084/182150523-23baf26f-b365-4173-a96c-d664fbd19987.png)

![image](https://user-images.githubusercontent.com/1331084/182149455-451a22e0-2f88-4d2b-b2f4-305ccb6ff85e.png)

Am i missing a step somewhere in the deployment for importing ES templates, or is this an actual bug needing some remedy in the Kibana dashboards?

**Comment by dougburks** [comment]
SIP and SSH visualizations appear to be working correctly for me. Have you verified that you have data with those fields?
![Screen Shot 2022-08-01 at 1 15 25 PM](https://user-images.githubusercontent.com/1659467/182205731-34308cd2-4cc8-461f-b071-5a7ac58a7811.png)

![image](https://user-images.githubusercontent.com/1659467/182206198-14bfcd5a-2fce-429c-945e-9e2d3d51096d.png)

**Comment by dougburks** [comment]
> Finding the new SOC thing to be far less useful for domain-specific hunting activities

We just released a new video on SOC Dashboards that you might want to check out:
https://youtu.be/xUBhyF7se8s
